vro-types/vrotsc-annotations/index.d.ts

```diff
@@ -245,6 +247,26 @@ interface VroWorkflowItemMethodDecorator {
 }
 
 
+// ---------------------------------------------- Workflow Canvas End Item ------------------------------------------------
+
+export declare const WorkflowEndItem: VroWorkflowEndItemDecorator;
+
+interface VroWorkflowEndItemDecorator {
+	(obj?: VroWorkflowEndItemConfiguration): VroWorkflowEndItemMethodDecorator;
+	new(obj?: VroWorkflowEndItemConfiguration): VroWorkflowEndItemConfiguration;
+}
+
+interface VroWorkflowEndItemConfiguration {
+	endMode?: WorkflowEndMode;
+	exception?: string;
```

Comment:
`exception` will mean sth else in other items... can we rename this?

Reply:
Code updated.